client/src/api/index.js

```diff
@@ -44,11 +52,46 @@ const request = async (url, { headers = {}, ...options } = {}) => {
   }
 }
 
+export const loginUser = () => {}
+// return array of api responses
```

Comment:
What's this comment about?

Reply:
A note for myself based on something David was explaining to me.

Reply:
Removed.